modules/recommended-scale-practices.adoc

```diff
@@ -26,3 +26,7 @@ handle impacts the process. The controller will start to query more while trying
 to create, check, and update the machines with the status. The cloud platform on
 which {product-title} is deployed has API request limits and excessive queries
 might lead to machine creation failures due to cloud platform limitations.
+
+Enable machine health checks when scaling to large node counts. In case of failures, 
+health checks monitors the condition and automatically repairs unhealthy machines in 
```

Comment:
the health checks monitor the condition and automatically repair

Reply:
@ahardin-rh fixed it, thanks.